### PR TITLE
James development

### PR DESCRIPTION
**Pull request by mrjwashiv**
Created the portfolio website. Didn't add much, but hopefully it gets the job done. I wanted to do the navigation bar, and thats what took up most of my time.

### CODE REVIEW

File: portfolio/src/main/webapp/foods.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <li id="navlink"><a href="hobbies.html">Hobbies</a></li>
+      </ul>
+      <h2>These are the foods I really love eating:</h2>
+      <ol>
+          <li>Nachos</li>
+          <li>French Fries</li>
+          <li>Chicken Tenders</li>
+          <li>Pizza</li>
+      </ol>
+      <img src="images/Nachos.jpg" alt="Picture of Nachos"
```
Comment (jayspringfield) [inline portfolio/src/main/webapp/foods.html:21]: now I want some nachos

File: portfolio/src/main/webapp/hobbies.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  </head>
+  <body>
+      <h1>My Hobbies</h1>
+      <ul id="navbar">
+        <li id="navlink"><a href="index.html">Home</a></li>
+        <li id="navlink"><a href="foods.html">Favorite Foods</a></li>
+      </ul>
+      <h2>These are some of my favorite things to do:</h2>
+      <ul>
+          <li>Watching Anime(alot)</li>
```
Comment (jayspringfield) [inline portfolio/src/main/webapp/hobbies.html:16]: tiny thing, should be (a lot)

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,10 +8,29 @@
...
+      <h1>James Washington's Portfolio</h1>
+      <ul id="navbar">
+          <li id="navlink"><a href="index.html">Home</a></li>
+          <li id="navlink"><a href="hobbies.html">Hobbies</a></li>
+          <li id="navlink"><a href="foods.html">Favorite Foods</a></li>
+      </ul>
+      <img src="images/ProfilePic.jpg" alt="My Profile Picture" width="300" height="350">
+      <table style="width:80%.">
+          <tr>
+              <td> This is my portfolio. It'll include a lot of things about me.
```
Comment (jayspringfield) [inline portfolio/src/main/webapp/index.html:20]: Can you fix the spacing so it's even?

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -8,10 +8,29 @@
...
+                    I'm born and raised in Kansas City, Missouri (I'm really from
+         	        a small town called Grandview, but it's close enough!). I have
+         	        3 siblings, of which I'm the oldest son. I have over 40 cousins,
+         	        so family gatherings are always a ton of fun! Education is big in
+         	        my family, so college is always pushed. I'm a rising junior and I
+         	        attend Morehouse College. I study Computer Science, and I plan on
+         	        going into industry when I graduate. I interned at Google Summer 2019,
+         	        and I'm interning Summer 2020 as well. Outside of these I don't have very
+         	        many projects.</th>
```
Comment (jayspringfield) [inline portfolio/src/main/webapp/index.html:29]: I think this is supposed to be a </td>. It should also be on its own line indented evenly with the opening <td>

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -7,3 +7,46 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+table, th, td {
+  border: 3px solid black;
+  border-collapse:collapse;
```
Comment (jayspringfield) [inline portfolio/src/main/webapp/style.css:13]: space after the :

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -7,3 +7,46 @@
...
+#navbar {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #333;
+}
+
+#navlink {
+  float:left;
```
Comment (jayspringfield) [inline portfolio/src/main/webapp/style.css:33]: space after the :

File: portfolio/src/main/webapp/foods.html (hunk 1 above)
Comment (mrjwashiv) [reply portfolio/src/main/webapp/foods.html:21]: Ikr! Might make some today.

**Pull request merged**